src/drivers/cmakeFileApiDriver.ts

```diff
@@ -245,11 +253,21 @@ export class CMakeFileApiDriver extends CMakeDriver {
             log.debug(`Configuring using ${this.useCMakePresets ? 'preset' : 'kit'}`);
             log.debug('Invoking CMake', cmake, 'with arguments', JSON.stringify(args));
             const env = await this.getConfigureEnvironment(configurePreset, options?.environment);
+
             const child = this.executeCommand(cmake, args, outputConsumer, {
                 environment: env,
                 cwd: options?.cwd ?? binaryDir
             });
             this.configureProcess = child;
+
+            // TODO: Before PR'ing and merging, I should add implementation to invoke cmake under the hood with the "-E capabilities" flag
```

Comment:
Will this be addressed?

Reply:
Ah, good catch, I've already implemented this, I'll remove.

Reply:
It looks like you did something in cmakeExecutable.ts already. Maybe make it available here?